backend/src/main/java/de/lindatroesken/backend/controller/MangelController.java

```diff
@@ -78,13 +79,18 @@ public MangelController(MangelService mangelService) {
     private MangelEntity map(Mangel mangel){
         return MangelEntity.builder()
                 .description(mangel.getDescription())
+                .category(mangel.getCategory())
+                .status(Status.valueOf(mangel.getStatus()))
```

Comment:
reicht da nicht mangel.getStatus()   ?

Reply:
nein. Ich mache aus einem String (im Dto) ein Enum (in der Entity)